financialaid/models.py

```diff
@@ -10,20 +9,42 @@
 from jsonfield import JSONField
 
 from courses.models import (
-    CourseRun,
     Program,
 )
 
 
-class Tier(models.Model):
+class TimeStampedModel(models.Model):
+    """
+    Base model for create/update timestamps
+    """
+    created_on = models.DateTimeField(auto_now_add=True)  # UTC
+    updated_on = models.DateTimeField(auto_now=True)  # UTC
+
+    def update(self, **kwargs):
```

Comment:
Ok, take out the `update` method for now, feel free to leave the abstract base class for timestamps